tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -60,6 +60,19 @@ public function testCodeWithoutReturnTypes($expected, $input = null)
         $this->doTest($expected, $input);
     }
 
+    /**
+     * @requires PHP 7.0
```

Comment:
Not needed because currently the minimum supported PHP version is 7.1 on this branch.